src/core/qgsfield.cpp

```diff
@@ -92,6 +92,15 @@ QString QgsField::displayName() const
     return d->name;
 }
 
+QString QgsField::displayNameWithAlias() const
+{
+  if ( alias().isEmpty() )
+  {
+    return name();
+  }
+  return QStringLiteral( "%1 (%2)" ).arg( name() ).arg( alias() );
```

Comment:
I always get curious about this -- do brackets need to be translated? Or are they universal?

Reply:
I've no idea :)